### PR TITLE
Misc translation 'ts' fixes

### PR DESCRIPTION
**Pull request by mlutfy**
Overview
----------------------------------------

Mostly NFC, this PR fixes a few places where the `ts` was incorrectly used. When incorrectly used, the strings are not extracted, not sent to Transifex, and probably not translated in the UI (in most cases).

The main exception might be the `ts` removal in `CRM/Contact/Form/Search/Criteria.php` or in `CRM/Contact/Form/Task/AddToParentClass.php `, but it's a hack, a bad example, and if they cause problems, they should be addressed correctly.

### CODE REVIEW

File: CRM/Core/Form/Task/Batch.php [hunk 1]
```diff
@@ -133,7 +133,7 @@ public function buildQuickForm() {
     $this->addButtons([
       [
         'type' => 'submit',
-        'name' => ts('Update ' . ucfirst($this::$entityShortname) . 's)'),
+        'name' => ts('Update'),
```
Comment (mattwire) [inline CRM/Core/Form/Task/Batch.php]: So this changes it from eg. "Update Membership" to "Update" - I think that's ok

File: CRM/Core/Payment/Form.php [hunk 2]
```diff
@@ -319,7 +319,7 @@ public static function validateCreditCard($values, &$errors, $processorID = NULL
       if (!empty($values['credit_card_type'])) {
         $processorCards = CRM_Financial_BAO_PaymentProcessor::getCreditCards($processorID);
         if (!empty($processorCards) && !in_array($values['credit_card_type'], $processorCards)) {
-          $errors['credit_card_type'] = ts('This procesor does not support credit card type ' . $values['credit_card_type']);
+          $errors['credit_card_type'] = ts('This procesor does not support credit card type %1', [1 => $values['credit_card_type']]);
```
Comment (mattwire) [inline CRM/Core/Payment/Form.php]: @bgm As we are touching this string can we correct the spelling: procesor => processor

**Review by mattwire — COMMENTED** [review]
All good @bgm There's one spelling it'd be nice to fix before merge

File: CRM/Core/Payment/Form.php (hunk 2 above)
Comment (mlutfy) [reply CRM/Core/Payment/Form.php]: @mattwire Good catch, I updated my PR.

File: CRM/Core/Form/Task/Batch.php (hunk 1 above)
Comment (mlutfy) [reply CRM/Core/Form/Task/Batch.php]: yep, fixing this was too complicated, and it seemed like the context was sufficient to understand what is being updated.

**Pull request merged**